__tests__/test-statusMatrix.js

```diff
@@ -394,4 +394,109 @@ describe('statusMatrix', () => {
       ['b/b.txt', 0, 2, 2],
     ])
   })
+
+  describe('statusMatrix with a symlinked directory', () => {
+    const realFolder = 'realFolder'
+    let storedProps = {}
+    const symlinkCreated = false
+    beforeAll(async () => {
+      const { fs, dir, gitdir } = await makeFixture('test-empty')
+      await fs.mkdir(path.join(dir, realFolder))
+      await fs.write(path.join(dir, realFolder, 'a.txt'), 'a')
+      // symlinks can't be created in browser, according to
+      // https://github.com/isomorphic-git/isomorphic-git/blob/main/__tests__/test-add.js#L117      symlinkCreated = await fs
+      const symlinkCreated = await fs
```

Comment:
This variable is const and it's not used in tests, If think it suppose to be without declaration and outside it should be declared as `let`.